test-nifikop-operator/deploy/values-laphets.yaml

```diff
@@ -0,0 +1,47 @@
+## NiFi Operator Image
```

Comment:
I guess this file is automatically generated. Could you gitignore such auto-generated files?

Reply:
The rule is hard to determine in gitignore, but I can delete it after it is used.

Reply:
@laphets 
Maybe sth like this https://github.com/sieve-project/sieve/blob/a275cf81af7707ea9c77d6ad382662a5a04e4263/.gitignore#L14?